### PR TITLE
Add support for builds db

### PR DESCRIPTION
**Pull request by SCdF**
# Description

Adds support for pushing builds in the style horticulturalist consumes. That is to say: a pure design doc. 

I want to get this merged ASAP so we can start generating builds with this system. This PR doesn't disable the current markets system, though we would remove support for that eventually.

medic/medic-webapp#3582

# Review checklist

- [x] Readable: Concise, well named, follows the [style guide](https://github.com/medic/medic-docs/blob/master/development/style-guide.md), documented if necessary.
- [x] Documented: Announced in Changes.md in plain English. Configuration and user documentation on [medic-docs](https://github.com/medic/medic-docs/)
- [x] Tested: Unit and/or e2e where appropriate
- [x] Internationalised: All user facing text
- [x] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in Changes.md.

### CODE REVIEW

File: scripts/ci/after_success.sh [hunk 1]
```diff
@@ -13,8 +13,17 @@ if [ -z "$UPLOAD_URL" ]; then
     exit 1;
 fi
 
-cd sentinel && npm install --production && cd .. && \
-cd api && npm install --production && cd ..
+# Attempt to reduce size of api and sentinel artifacts
+(cd sentinel &&
```
Comment (mandric) [inline scripts/ci/after_success.sh]: why the subshell/parens? so you don't need to manage the current working directory?  does it fail properly with the `set -e`?

File: scripts/ci/after_success.sh [hunk 2]
```diff
@@ -66,4 +77,18 @@ elif [[ "$TRAVIS_TAG" =~ ^[0-9]+\.[0-9]+\.[0-9]+-rc\.[0-9]+$ ]]; then
 
 fi;
 
+### ---- TODO delete this section once we have fully moved to horticultularist END ---- ###
+
+echo 'Building build for builds database...'
```
Comment (mandric) [inline scripts/ci/after_success.sh]: Maybe "Push to `builds` database on staging..." is more clear?

File: scripts/ci/after_success.sh [hunk 3]
```diff
@@ -66,4 +77,18 @@ elif [[ "$TRAVIS_TAG" =~ ^[0-9]+\.[0-9]+\.[0-9]+-rc\.[0-9]+$ ]]; then
...
+if [[ -n "$TRAVIS_TAG" ]]; then
+    node --stack_size=10000 `which kanso` push --minify \
```
Comment (mandric) [inline scripts/ci/after_success.sh]: I don't see why you wouldn't use the push function with the retry in it?

**Comment by alxndrsn** [comment]
Does this duplicate functionality from https://github.com/medic/medic-webapp/pull/3560?  There are some relevant TODO items on that PR which might also apply to this one.

**Comment by SCdF** [comment]
@alxndrsn 100% I wanted to get some relevant part of it merged. If you think that PR is going to be merge-ready any time soon then this one can be dropped

**Comment by SCdF** [comment]
Happy with and have reviewed the changes @alxndrsn made on this to split them up.

**Pull request merged**